src/components/Navbar.jsx

```diff
@@ -49,50 +49,41 @@ const Navbar = () => {
             <li>
               <NavLink
                 to="/"
-                className={`inline-block py-2 px-3 text-center font-bold  rounded-md ${
-                  theme.mode === "light"
-                    ? "hover:text-white hover:bg-black focus:text-white focus:bg-black"
-                    : "hover:text-black hover:bg-white focus:text-black focus:bg-white"
-                }`}
-                activestyle={{
-                  fontWeight: "bold",
-                  color: navbar.color,
-                  backgroundColor: "white",
-                }}
+                className={({ isActive }) =>
+                  `inline-block py-2 px-3 text-center font-bold  rounded-md ${
+                    theme.mode === "light"
+                      ? `hover:text-white hover:bg-black ${isActive && "text-white bg-black"}`
+                      : `hover:text-black hover:bg-white ${isActive && "text-black bg-white"}`
+                  }`
+                }
               >
                 Home
               </NavLink>
             </li>
             <li>
               <NavLink
                 to="/ProjectsPage"
```

Comment:
change to the `projectspage`
```suggestion
                to="/projectspage"
```